proto/kvrpcpb.proto

```diff
@@ -716,6 +716,9 @@ message Context {
 
     // The informantion to trace a request sent to TiKV.
     tracepb.TraceContext trace_context = 23;
+
+    // The source of the request, will be used as the tag of the trace.
+    string request_source = 24;
```

Comment:
If I understand it correctly, it's not used for tracing. Instead, it will be used as the label of some metrics reporting to prometheus.

So I'm hesitating about the field name. I'm a bit more inclined to use a more ambiguous word like "request_label" and it represents its real function better.

Reply:
The word "label" is a bit common, Prometheus metrics already use "label" for metrics classification, we have "labelType", "labelName", etc. If we name it "request_label", and add "labelLabel" for some metrics, that would be confusing.

Reply:
I think "source" is okay. We can hint in the comment that this is essentially an extra label in metrics and the user may use it for any actual meaning.